thanos-operator.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: thanos-operator
+  version: 0.3.7
+  epoch: 0
+  description: Kubernetes operator for deploying Thanos
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/banzaicloud/thanos-operator
+      tag: ${{package.version}}
+      expected-commit: 7b87b4f89dee6454ae46e9579b9a98d6d7a5e550
+
+  - runs: |
+      go build -o bin/manager main.go
```

Comment:
```Suggestion 
      # Address  CVE-2022-21698  GHSA-69ch-w2m2-3vjp  GHSA-69cg-p879-7622
      go get github.com/prometheus/client_golang@v1.11.1
      go get golang.org/x/text@v0.3.8
      go get golang.org/x/net@v0.7.0
      
      go build -o bin/manager main.go
```

Reply:
thanks so much for this! I was also planning to do the same thing, but you beat me!